src/main/java/ch/uzh/ifi/hase/soprafs22/rest/mapper/DTOMapper.java

```diff
@@ -34,7 +34,14 @@ public abstract class DTOMapper {
     @Mapping(source = "team", target = "team")
     public abstract PlayerGetDTO convertIPlayerToPlayerGetDTO(IPlayer player);
 
-
+    //add invitation code if token matches
+    public LobbyGetDTO convertILobbyToLobbyGetDTO(ILobby lobby, String token) {
+        LobbyGetDTO ret = convertILobbyToLobbyGetDTO(lobby);
```

Comment:
can you think of a more descriptive name than "ret"?

Reply:
sure, done.